commit ec55d9450a211822c32a00d8ff12bcc8534d9efe (refs/changes/41/3541/2)
Author: Orlando Vazquez <ovazquez@gmail.com>
Date:   2018-03-06T12:05:14-08:00 (1 year, 7 months ago)
    
    TRITON-207 Disable sdc-docker copy out of stopped container test due to OS-6699
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/test/integration/cli-copy.test.js b/test/integration/cli-copy.test.js
index 4f8a2e0..7aa2c77 100644
--- a/test/integration/cli-copy.test.js
+++ b/test/integration/cli-copy.test.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -417,7 +417,9 @@ test('copy a file out of running container', function (tt) {
 });
 
 
-test('copy a file out of stopped container', function (tt) {
+test('copy a file out of stopped container (skipped until OS-6699 fixed)',
+{ skip: true },
+function testCopyOutOfStoppedContainer(tt) {
     tt.plan(9);
 
     var fnbase = '/var/tmp';
